lib/faraday/adapter/net_http_persistent.rb

```diff
@@ -193,7 +194,7 @@ def configure_ssl(http, ssl)
         http_set(http, :cert_store, ssl_cert_store(ssl))
 
         SSL_CONFIGURATIONS
-          .select { |_, key| ssl[key] }
+          .select { |_, key| ssl[key] != nil }
```

Comment:
With the previous logic both `nil` and `false` will result in the option being ignored and the default used, that is wrong.

Reply:
Nice fix!

Reply:
Oops, fixed the linter error!